docs/guides/live.md

```diff
@@ -0,0 +1,105 @@
+# The live user interface and API in Video.js
+> Note the the "old" live user interface is currently the default, see the section on [the new user interface](#the-new-user-interface) for information on setting that up.
+
+
+## The default live user interface
+The default user interface hides the `ProgressControl` component on the controlbar and shows the `LiveDisplay` component when Video.js detects that the video that it is playing is live (via a `durationchange` event).
```

Comment:
the check is actually whether the duration is infinite, (or not -1). It just happens during a `durationchange` event.

Reply:
There's also special handling for Android but we don't need to mention that here.